.github/actions/pr-chart/action.yaml

```diff
@@ -148,8 +198,53 @@ runs:
       run: |
         helm push ${{ inputs.registry-repo }}-${{ env.PR_COMMIT_TAG }}.tgz oci://${{ env.CHART_REPO }}
 
+    - name: create step summary on push success
+      shell: bash
+      run: |
+        echo "${SUMMARY}" >> "$GITHUB_STEP_SUMMARY"
+      env:
+        SUMMARY: |
+          ```
+          ${{ inputs.registry }}/${{ inputs.registry-project }}/chart/${{ inputs.registry-repo }}:0.0.0-pr.${{ github.event.pull_request.number }}+${{ env.GITHUB_SHORT_SHA }}
+          ```
+          :hammer_and_wrench: Push Success  
+
+    - name: create step summary on failure
+      if: failure()
```

Comment:
This needs to be `failure() && ...` to test if the outcome of L191 was a failure.